src/components/ManaCalculator/hooks/useManaState.ts

```diff
@@ -186,9 +186,19 @@ export function useManaState() {
 export function getDefaultParameters(
   network: NetworkType,
 ): ManaCalculatorProps {
+  const networkParams = {
+    [NetworkType.IOTA]: {
+      initialEpoch: 1,
+      finalEpoch: 365,
+    },
+    [NetworkType.SHIMMER]: {
+      initialEpoch: 0,
+      finalEpoch: 100,
```

Comment:
```suggestion
      initialEpoch: 1,
      finalEpoch: 365,
```

Reply:
I think we decided to default it to 1000 actually